integration-tests/messaging/jms/src/main/java/org/apache/camel/quarkus/messaging/jms/JmsRoutes.java

```diff
@@ -44,11 +47,13 @@ public void configure() throws Exception {
                 .bean("destinationHeaderSetter")
                 .toF("%s:queue:override", componentScheme);
 
-        fromF("%s:queue:xa", componentScheme)
+        fromF("%s:queue:xa?transactionManager=#jtaTransactionManager", componentScheme)
```

Comment:
Why only this one needs to bind transaction manager and not others?

Reply:
Please refer to `camel-jms` document https://camel.apache.org/components/4.0.x/jms-component.html#_enabling_transacted_consumption

And for `JtaTransactionManager`, the propert `transacted=true` is not needed because the transaction is managed by the container like `quarkus-arc`.

Reply:
Also it looks like only restrict with the IBM MQ client. Without the `transactionManager` property, it throws `MQRC_SYNCPOINT_NOT_AVAILABLE` error when starting to consume from a queue.

I test it with `artemis-jms-client` and it can work without this property.